Source/animation/Easing.swift

```diff
@@ -12,4 +12,5 @@ public enum Easing {
     case easeIn
     case easeOut
     case easeInOut
+    case spring(mass: CGFloat, stiffness: CGFloat, damping: CGFloat, initialVelocity: CGFloat)
```

Comment:
`CGFloat` => `Double`. Does it make sense to add default values?

Reply:
it does, but I can't - enum doesn't allow that

Reply:
So may be it's better to replace `Easing` with a class then?)

Reply:
It will change public API, I know a guy who won't allow it)

Reply:
No, it's not. We did the same for `Align` and `AspectRatio` already.